ui/components/MeshConfigComponent.js

```diff
@@ -553,63 +552,82 @@ handleNATSClick = () => {
   };
 
   handleResetDatabase = () => {
-    this.props.updateProgress({ showProgress : true });
-    const self = this;
-    resetDatabase({
-      selector : {
-        clearDB : "true",
-        ReSync : "true",
-        HardReset : "true"
-
-      },
-      // For now by default set to hard reset.
-      // TODO: User should be able to select soft or hard reset when "Reset Daatabase" btn is clicked.
-    }).subscribe({
-      next : (res) => {
-        self.props.updateProgress({ showProgress : false });
-        if (res.resetStatus === "PROCESSING") {
-          this.props.enqueueSnackbar(`Database reset successful.`, {
-            variant : "success",
-            action : (key) => (
-              <IconButton key="close" aria-label="close" color="inherit"  onClick={() => self.props.closeSnackbar(key)}>
-                <CloseIcon />
-              </IconButton>
-            ),
-
-            autohideduration : 2000,
-          })
-        }
-      },
-      error : self.handleError("Database is not reachable, try restarting server.")
-    });
+    return async () => {
+      const modalofRefOfResetDatabase = this.modalRefOfResetDatabase.current;
+      let responseOfResetDatabase = await modalofRefOfResetDatabase.show({
+        title : "Reset Database?",
+        subtitle : "Are you sure to reset all the database of the meshery?",
+        options : ["yes", "no"]
+      });
+      if (responseOfResetDatabase === "yes") {
+        this.props.updateProgress({ showProgress : true });
+        const self = this;
+        resetDatabase({
+          selector : {
+            clearDB : "true",
+            ReSync : "true", // True - For Hard reset, False otherwise
+            hardReset : "true",
+          },
+          // For now by default set to hard reset.
+          // TODO: User should be able to select soft or hard reset when "Reset Daatabase" btn is clicked.
```

Comment:
Can you please remove these comments?? No longer needed.

Reply:
ok :+1: